args/core/platform/platform.hpp

```diff
@@ -9,8 +9,45 @@
  */
 #define ARGS_CPP17V 201703L
 
+#if defined(_DEBUG)
```

Comment:
note that on linux compilers it is common to define DEBUG and NDEBUG instead of _DEBUG. (NDEBUG=Release).
```suggestion
#if defined(_DEBUG) || defined(DEBUG)
```